prep-javascript-objects/main.js

```diff
@@ -0,0 +1,21 @@
+var person = {
+  firstName: 'Bingle',
+  lastName: 'Padilla',
+  hobby: 'Cooking'
+};
+
+console.log(person);
+
+var firstName = 'Bingle';
+var lastName = ' Padilla';
+var fullName = firstName + lastName;
+console.log("The person's name is:", fullName);
```

Comment:
You can use dot notation to retrieve the values of an object's properties.

```js
var fullName = person.firstName + ' ' + person.lastName;
```

Reply:
![prep-javascript-objects-part2](https://user-images.githubusercontent.com/90303824/149820931-a19c1f00-deed-4604-83ea-5fecb36d2325.png)
![prep-javascript-objects-mainjs-file](https://user-images.githubusercontent.com/90303824/149820956-1441155d-3885-4d90-973c-1206beb1e371.png)

I have updated the file.